test/addons/rook-cephfs/snapshot-class.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# yamllint disable-line rule:line-length
+# Source: https://raw.githubusercontent.com/rook/rook/release-1.15/deploy/examples/csi/cephfs/snapshotclass.yaml
+# Modifications:
+#  - Added storageID and replicationID
+---
+apiVersion: snapshot.storage.k8s.io/v1
+deletionPolicy: Delete
+driver: rook-ceph.cephfs.csi.ceph.com
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-cephfsplugin-snapclass
+  labels:
+    ramendr.openshift.io/storageid: rook-cephfs-$cluster-1
+    ramendr.openshift.io/replicationid: rook-cephfs-$cluster-1
```

Comment:
No need to add repID to SnapClass, we need to match snap class on the local cluster to the StorageClass, and not across clusters.

Reply:
i have removed the replicationID from here.